data/Rajeshds.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "D S Rajesh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "An enthusiastic Tech Student. I develop softwares and websites, currently working on Full Stack Web Development.",
+  "avatar": "https://github.com/rajeshds20.png",
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/devangamsajjarajesh",
+      "icon": "FaLinkedin" 
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/Rajeshds20",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/rajeshds55",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/rajeshds20",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "HashNode",
+      "url": "https://rajeshds.hashnode.dev",
+      "icon": "SiHashnode"
+    },
+    {
+      "name": "Medium",
+      "url": "https://rajeshds.medium.com",
+      "icon": "FaMedium"
+    },
+    {
+      "name": "Dev Community",
+      "url": "https://dev.to/rajeshds",
+      "icon": "FaDev"
+    },
+    {
+      "name": "HackerRank",
+      "url": "https://www.hackerrank.com/dsrajesh71",
+      "icon": "FaHackerrank"
+    },
+    {
+      "name": "Codechef",
+      "url": "https://www.codechef.com/users/rajeshds",
+      "icon": "SiCodechef"
+    },
+    {
+      "name": "Email",
+      "url": "dsrajesh71@gmail.com",
```

Comment:
one more thing, This should be a `mailto` link here
change this URL to `mailto:dsrajesh71@gmail.com`